network/src/lib.rs

```diff
@@ -7,6 +7,9 @@
 // </Black magic>
 
 // Public exports
+#[macro_use]
+extern crate prometheus;
+
```

Comment:
Please don't do this, you shouldn't use "extern crate" or "#[macro_use]" anymore

Reply:
Looks like prometheus doesn't have a release which is 2018 compatible yet...sorry for the noise

Reply:
@bmwill Sorry for introducing more of this, how should we followup on fixing in the future? Do we open an issue to deprecate "extern crate" or "#[macro_use]" when prometheus release a new version?

Reply:
Yeah we could open an issue and hopefully they do another release soon :)